low-level/files/iec_mapping/ha

```diff
@@ -1,2 +1,4 @@
 0080010001,NodeFailure,The cluster has lost one server. System is running in degraded mode.
-0080020001,NodeRecovery,Server has joined back the cluster. System is restored.
\ No newline at end of file
+0080020001,NodeRecovery,Server has joined back the cluster. System is restored.
+0080030001,IOServiceStop,IO services are getting restored on current server.
+0080030002,IOServiceStart,IO services are failing on current server due to hardware or service failure.
```

Comment:
Description of start/stop seems confusing e.g. "IOServiceStop,IO services are getting restored on current server". Is it right?

Reply:
Updated msg